maxim/tests/test_gemini.py

```
@@ -1,39 +1,27 @@
 import os
 import unittest
 from uuid import uuid4
 
+import dotenv
 from google import genai
 from google.genai.types import Content, Part
 
-from maxim import Config, Maxim
+from maxim import Maxim
 from maxim.logger import (
     GenerationConfig,
```

Ruff (0.11.9)

`maxim.logger.GenerationConfig` imported but unused

Remove unused import

(F401)